uffizzi/docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,58 @@
+version: '3'
+
+x-uffizzi:
+  ingress:
+    service: nginx
+    port: 80
+
+services:
+  app:
+    image: "${APP_IMAGE}"
+    restart: unless-stopped
+    working_dir: /var/www/
+    command: ["-c","
```

Comment:
Does this work on Uffizzi?

Reply:
Yeah, looks like it works. I don't have even an `entrypoint` in the docker file. :)

https://github.com/daramayis/f-crater/blob/a84319274a13a538a60f585ab1448e6de51e50e7/uffizzi/Dockerfile